guide/web-streaming.md

```diff
@@ -15,6 +15,19 @@ except that the `X3D` file and the animation are sent on the fly to the Web brow
 
 %end
 
+The web interface displays a toolbar with the following actions:
```

Comment:
No, I really mean actions like play, pause, revert the simulation, etc.
But I will change it with items.